lerna.json

```diff
@@ -1,7 +1,8 @@
 {
-  "lerna": "2.7.1",
+  "lerna": "2.11.0",
```

Comment:
We can not yet update to lerna 3.x because of `cz-lerna-changelog` [not being updated](https://github.com/atlassian/cz-lerna-changelog/issues/16)

Reply:
Hmm, why are we still using that?

Reply:
It's used by `commitizen` when running the commit script.

https://github.com/commercetools/nodejs/blob/083e5590c50f28b5d50dc0d904903aad3a7ccb21/package.json#L15

Reply:
I would get rid of it. I think no one is using that command...?

Reply:
Sounds good, will do. I'll keep commitizen itself tho as it's amazing for on-boarding students and introducing them to angular commit convention.

Reply:
Ask around in the team. I am not aware of anyone using it anymore. It was good introducing the style but should not be needed anymore. I would recommend removing it in amothrr PR.

Reply:
commitizen itself is so drilled into my spine that it is part of my muscle memory now.
However, if you feel that you want to keep this to ease "onboarding", or in some extent ensuring consistency in commit messages, then don't listen to us 👍